### DISCUSSION TITLE
Multiple services under the same umbrella

### DISCUSSION THREAD

**Discussion by ohreally** [tosdr/edit.tosdr.org#1000] [Ideas]
Hi,

It would be nice if multiple services could be linked to each other.

As an example: Stack Overflow and Stack Exchange.
Stack Overflow has an E rating and Stack Exchange has an A rating. Yet they are both served under the same terms. And I don't mean similar terms; I mean that the '_Privacy Policy_' link at Stack Exchange directs to the privacy policy at Stack Overflow, and that the title of that privacy policy at Stack Overflow is '_Stack Exchange, Inc. Privacy Policy_'.

So it would be nice if we could say _SiteA=SiteB_. Not only would this make sure that both are equally up-to-date, it would also avoid people investing time and energy in the review of terms that have already been reviewed.

Thanks for considering,
  Rob

**Comment by michielbdejong** [comment]
Yes! Thanks for reporting!!

Our general rule is if the document URLs are identical then we count it as one service. So in this case those two should definitely be merged.

Oh wow, especially if the ratings diverge so much, that's a big problem.

We don't have a good automated process for this yet, Maybe the easiest would be someone updates the 'Service' on all the points of one of them to the other, and then we deprecate the one that's left empty.

I think I'll move all the points from https://edit.tosdr.org/services/312 (StackOverflow) to 
https://edit.tosdr.org/services/300 (StackExchange).

**Comment by michielbdejong** [comment]
It'll be a bit of a complex migration though, because if I just update the `service_id` in the Points table from 312 to 300 then the quotes will point to the wrong doc excerpts... Will think about it a bit before trying this :)

**Comment by ohreally** [comment]
And while thinking about this, please also read the comments I made at https://edit.tosdr.org/services/312 (this service is also the same as superuser.com, serverfault.com, askubuntu.com, stackapps.com and mathoverflow.net).

Thanks!

**Comment by ohreally** [comment]
And, just curious...
If you just move all points to a single service, will the Firefox plugin still recognize both services?

**Comment by ohreally** [comment]
Haven't looked at all the ins and outs (would love to, but really don't have the time at the moment), but wouldn't the simplest solution just be to create a new db table 'service_aliases', linked to the services table? Where necessary db queries could then be changed to joins, and you're done. This way you could link unlimited numbers of services, and also easily change links when e.g. a company sells part of their services.
(Obviously I could be missing things; I'm just trying to put in the 2 minutes I can spare each day.)

**Comment by ghost** [comment]
This raises the question of services like YouTube Kids. It refers back to policies like Google and main YouTube but also has its own set of Privacy Policies and Terms of Services that trump the original.
How can services like this be handled appropriatly?
